src/components/data/DataTable.js

```diff
@@ -178,7 +179,7 @@ const DataTable = props => {
             'aria-label': 'Search',
             placeholder: 'Search',
             onChange: v => {
-              setActiveTextFilter(v)
+              setActiveTextFilter(v.toString().trim())
```

Comment:
the `toString()` shouldn't be necessary here, but felt like safe defensive coding - we've seen other instances where we expected a string but got something else and caused page-killing errors